diff -Naur gobject-introspection-1.43.3.org/Makefile-tools.am gobject-introspection-1.43.3/Makefile-tools.am
--- gobject-introspection-1.43.3.org/Makefile-tools.am	2015-03-09 19:24:12.933415053 +0100
+++ gobject-introspection-1.43.3/Makefile-tools.am	2015-03-09 19:22:57.733698904 +0100
@@ -10,7 +10,7 @@
 	tools/g-ir-annotation-tool.in	\
 	tools/g-ir-doc-tool.in
 
-TOOL_SUBSTITUTIONS = sed -e s,@libdir\@,$(libdir), -e s,@datarootdir\@,$(datarootdir), -e s,@PYTHON\@,$(PYTHON),
+TOOL_SUBSTITUTIONS = sed -e "s,@libdir\@,$(libdir)," -e "s,@datarootdir\@,$(datarootdir)," -e "s,@PYTHON\@,$(PYTHON),"
 
 g-ir-scanner: tools/g-ir-scanner.in _giscanner.la Makefile
 	$(AM_V_GEN) $(TOOL_SUBSTITUTIONS) $< > $@.tmp && mv $@.tmp $@
