--- lua-5.3.4.orig/src/Makefile	2017-07-29 11:13:33.143342024 +0200
+++ lua-5.3.4/src/Makefile	2017-07-29 11:14:37.675406819 +0200
@@ -13,6 +13,7 @@
 
 AR= ar rcu
 RANLIB= ranlib
+STRIP= strip
 RM= rm -f
 
 SYSCFLAGS=
@@ -114,7 +115,7 @@
 
 mingw:
 	$(MAKE) "LUA_A=lua53.dll" "LUA_T=lua.exe" \
-	"AR=$(CC) -shared -Wl,--out-implib,liblua.dll.a -o" "RANLIB=strip --strip-unneeded" \
+	"AR=$(CC) -shared -Wl,--out-implib,liblua.dll.a -o" "RANLIB=$(STRIP) --strip-unneeded" \
 	"SYSCFLAGS=-DLUA_BUILD_AS_DLL" "SYSLIBS=" "SYSLDFLAGS=-s" lua.exe
 	$(MAKE) "LUAC_T=luac.exe" luac.exe
 
